Drupal 6 to Drupal 8 migration tests on SAX 2017.txt

```diff
@@ -0,0 +1,69 @@
+guifi.net DRUPAL 6 TO DRUPAL 7 MIGRATION
+setup dev environment on https://github.com/guifi/docker-drupal-guifi
+edit /etc/hosts and add 127.0.0.1	guifi.dev (no problem with that since RewriteBase is commented on /usr/share/drupal/guifi-web/.htaccess)
+$db_url = 'mysql://username:password@localhost/databasename'; should be commented on On line 91 
+
+go to http://guifi.dev:8080/user and login as webmestre
+IT IS ALWAYS RECOMMENDED TO DISABLE AND UNINSTALL ALL MODULES THAT YOU DON'T NEED. P.E. Throttle module. I'm not going to do it since I don't know them!!!
+change user's email as it's conflicting with all the other user's because docker script
+do a docker ps
+do a docker exec -it <ID> bash
```

Comment:
with only docker-compose exec guifi-drupal /bin/bash it works (no hash look up needed)